app/Jobs/CreateCallPowerPostInRogue.php

```diff
@@ -43,32 +43,24 @@ public function handle(Rogue $rogue)
         // Using the callpower_campaign_id, get the action id from Rogue.
         $actionId = $rogue->getActionIdFromCallPowerCampaignId($this->parameters['callpower_campaign_id']);
 
-        // Check if the post exists in Rogue. If not, create the post.
-        $existingPost = $rogue->getPost([
-          'action_id' => $actionId,
-          'northstar_id' => $user->id,
+        info('creating post in rogue for northstar user: ' . $user->id);
```

Comment:
Would it be helpful if this log included that this was coming from the CallPower job?

Reply:
@katiecrane added the details!